cloudant.js

```diff
@@ -92,9 +92,11 @@ function Cloudant(options, callback) {
    // https://console.bluemix.net/docs/services/Cloudant/api/authorization.html#modifying-permissions
     var set_security = function(permissions, callback) { // eslint-disable-line camelcase
       var path = '_api/v2/db/' + encodeURIComponent(db) + '/_security';
-      return nano.request({ path: path,
+      return nano.request({
+        path: path,
         method: 'put',
-        body: {cloudant: permissions} }, callback);
```

Comment:
Is it worth doing something like this so people can get it right either way:
```js
if permissions.cloudant {
  body = permissions
} else {
  body = {cloudant: permissions}
}
```

I'm not sure we should add the complexity given that the doc makes it clear what kind of object is required, but am just thinking "aloud".

Reply:
Unsure. Would this still work with `couchdb_auth_only`? You'd need to specify a `cloudant` key I guess.